Packs/NutanixHypervisor/Integrations/NutanixHypervisor/NutanixHypervisor_description.md

```diff
@@ -0,0 +1,31 @@
+## Nutanix
+
+### Available Alert Type IDs, Impact Types, and Entity Types
```

Comment:
can you please post screenshot of this from the ui?

Reply:
fixed it
<img width="798" alt="Screen Shot 2021-02-03 at 17 14 13" src="https://user-images.githubusercontent.com/70005542/106767289-5df34c80-6643-11eb-8f19-a8a0193a87f7.png">